oat-sa/tao-core
# Feature/aut 2978/rdf import export scripts

Ticket: https://oat-sa.atlassian.net/browse/AUT-2978

A use case for this CLI scripts is to move RDF data like classes, resource definitions, metadata and a tree structure for tests, items, media assets keeping their original URI's, relations and data links, so that, by exporting/importing RDF and copying filesystem folders from one environment to another make authoring content quickly migrated even if code versions on environments are very different.

How to test:
- On env1 run export command for some RDF class or resource.
- Run Import command with the output file on env2
- Check that the migrated resource tree with it's metadata become available on target env
- Copy related FS files from env1 to env2 if needed and check that resources opening/previewing as well.
